Kernel/Arch/TrapFrame.cpp

```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright (c) 2018-2021, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2023, Idan Horowitz <idan.horowitz@serenityos.org>
```

Comment:
I'm sure this is GHs fault, but this looks weird
a file move with a license change?

Reply:
Yeah not sure how this happened.